internal/cmd/controller/gitops/reconciler/gitjob_controller.go

```diff
@@ -57,6 +57,8 @@ const (
 
 	defaultPollingSyncInterval = 15 * time.Second
 	gitPollingCondition        = "GitPolling"
+	generationLabel            = "generation"
```

Comment:
```suggestion
	generationLabel            = "fleet.cattle.io/gitrepo-generation"
```

I'd prefix with fleet and clarify what kind of generation is stored.